### DISCUSSION TITLE
Is there a keyboard shortcut to swap between Kanban and Markdown views?

### DISCUSSION THREAD

**Discussion by ddanieltan** [mgmeyers/obsidian-kanban#716] [Ideas & Feature Requests]
After going into Markdown view, the only way I can go back to Kanban view is to close and reopen. Is there some keyboard shortcut I'm missing?

**Comment by jowibou** [comment]
You can assign one in Hotkeys to the Command "Kanban: Toggle Between Kanban and Markdown Mode"
